src/app/components/SubjectHeading/SubjectHeading.jsx

```diff
@@ -230,12 +229,13 @@ class SubjectHeading extends React.Component {
           </td>
           <td className="subjectHeadingsTableCell subjectHeadingAttribute titles">{`${bib_count}`}</td>
           <td className="subjectHeadingsTableCell subjectHeadingAttribute narrower">{`${desc_count || '-'}`}</td>
+          {container !== 'context' &&
           <td className="subjectHeadingsTableCell sortButton">
-            { open && narrower.length > 1 && uuid.length > 0 && (container !== 'context')
+            { open && narrower.length > 1 && uuid.length > 0 && container !== 'context'
               ? <SortButton sortBy={sortBy} handler={this.sortHandler} />
               : null
             }
-          </td>
+          </td>}
         </tr>
         { open && narrower.length > 0 ?
           <SubjectHeadingsTableBody
```

Comment:
Hi @EdwinGuzman. Here is an example of where we are passing a lot of props down the tree because our approach to component structuring has not allowed us to accommodate a lot of nuance easily.